content/blog/en/announcing-dojo-2-beta-4.md

```diff
@@ -230,11 +230,11 @@ We have introduced two new widgets in Beta 4 and are considering more before the
 
 We have introduced a new accordion pane widget (`@dojo/widgets/accordionpane/AccordionPane`):
 
-![The accordion pane widget](/assets/blog/announcing-dojo-2-beta-4/accordion.png)
+![The accordion pane widget](assets/blog/announcing-dojo-2-beta-4/accordion.png)
```

Comment:
removed the leading `/` as resources will be loaded from the application base, which by default is `/`